src/content/get-started/transfer-a-sparsified-model/cv-object-detection.mdx

```diff
@@ -58,7 +58,7 @@ $ sparseml.yolov5.train -h
 With the sparsified model successfully trained, it is time to export it for inference.
 The `sparseml.yolov5.export_onnx` command is used to export the training graph to a performant inference one.
 After the command completes, a `model.onnx` file is created in `yolov5/sparsified` folder.
-It is now ready for deployment with the DeepSparse Engine utilizing its pipelines.
+It is now ready for deployment with DeepSparse utilizing pipelines.
```

Comment:
Since DeepSparse describes the suite of components, are you saying one can deploy with the whole suite? Are all four components doing the deployment?